.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "arrow-parens": ["error", "as-needed"]
+  },
+  "ignorePatterns": ["dist/", "build/"]
+}
```

Comment:
## Let's sort out the linter setup

- [ ] have an issue when running the project using 'npm start'. The local code is receiving this notice upon loading of the page. Let's relook how we set up the linter and address this.

![linter](https://user-images.githubusercontent.com/22345961/97418203-98996a80-1919-11eb-9c2a-764824d45891.png)